lms/models/lms_user.py

```diff
@@ -46,6 +46,11 @@ class LMSUser(CreatedUpdatedMixin, Base):
 
     display_name: Mapped[str | None] = mapped_column(index=True)
 
+    @property
+    def user_id(self) -> str:
+        """Alias lti_user_id to user_if for compatilbiity with models.User."""
```

Comment:
We could add a few `getattr`, `hasattr` elsewhere but this reduces that clutter a bit.